<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Configurator</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="proj_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project Path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="projdir"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_gamedir">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Targets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="target_win64">
        <property name="text">
         <string>Windows x64</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="target_linux64">
        <property name="text">
         <string>Linux x64</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Game Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Content folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="content_folder_name">
        <property name="text">
         <string>Content</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Engine folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="engine_folder_name">
        <property name="text">
         <string>Engine</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>API</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_api">
        <item>
         <property name="text">
          <string>OpenGL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Pipeline</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_pipeline">
        <item>
         <property name="text">
          <string>PBR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lambert</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Startup map</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="startup_map"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Base Resolution</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="res_w">
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>1270</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="res_h">
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>720</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Engine</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Engine Version</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_engine_ver"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="create_button">
       <property name="text">
        <string>Create Project !</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browse_gamedir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BrowseProjDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CreateProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BrowseProjDir()</slot>
  <slot>CreateProject()</slot>
  <slot>Cancel()</slot>
 </slots>
</ui>
